Bejebeje.Services/Services/ArtistsService.cs

```diff
@@ -27,7 +27,7 @@ public async Task<IList<ArtistCardViewModel>> GetArtistsAsync()
         FirstName = x.FirstName,
         LastName = x.LastName,
         Slug = x.Slugs.Where(y => y.IsPrimary).First().Name,
-        ImageUrl = x.ImageUrl
+        ImageId = x.Image == null ? 0 : x.Image.Id
```

Comment:
I'd use a `int?` to represent that. It's optional, defaulting to 0 isn't good imo, cause 0 might be a valid image Id.